src/Config.hs

```diff
@@ -110,6 +113,7 @@ parseEnvFlags envVars
   { efConnectTls = lookupEnvFlag "VAULTENV_CONNECT_TLS"
   , efValidateCerts = lookupEnvFlag "VAULTENV_VALIDATE_CERTS"
   , efInheritEnv = lookupEnvFlag "VAULTENV_INHERIT_ENV"
+  , efSearchPath = lookupEnvFlag "VAULTENV_SEARCH_PATH"
```

Comment:
So you would set this as `VAULTENV_SEARCH_PATH=true`, if you want it to search for the executable, that `vaultenv` should call, in the local PATH, right?

The name actually doesn't sound like a boolean. It sounds like we're setting a concrete path like e.g.
`VAULTENV_SEARCH_PATH=/some/path'.

OTOH I guess this is already true for all the other env variables we have, so we can also just leave it and stay consistent with how it is already.

Reply:
Oh right, that's true. Named it `VAULTENV_USE_PATH` after our offline discussion